packages/ibm-products/src/components/CreateTearsheet/CreateTearsheet.tsx

```diff
@@ -438,7 +438,7 @@ CreateTearsheet.propTypes = {
   /**
    * The main title of the tearsheet, displayed in the header area.
    */
-  title: PropTypes.node,
+  title: PropTypes.string,
```

Comment:
This should remain `ReactNode` from what I can tell. Sometimes people have more complex titles that require passing in a node rather than a string.